app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -255,7 +256,9 @@ def cb_enabled_value_for_nested
   def create_custom_button(input, model, record)
     button_id = input[:id]
     button_name = input[:name].to_s
-    record_id = if cb_send_checked_list
+    record_id = if cb_send_checked_list ||
```

Comment:
Can you replace this change with what's been done in https://github.com/ManageIQ/manageiq-ui-classic/pull/4887 please?

(It should be equivalent, but no point in not doing the same thing as on master.)